dll/win32/browseui/explorerband.cpp

```diff
@@ -193,19 +128,17 @@ void CExplorerBand::InitializeExplorerBand()
     NavigateToCurrentFolder();
 
     // Register shell notification
-    shcne.pidl = pidl;
-    shcne.fRecursive = TRUE;
-    m_shellRegID = SHChangeNotifyRegister(
-        m_hWnd,
-        SHCNRF_NewDelivery | SHCNRF_ShellLevel | SHCNRF_InterruptLevel | SHCNRF_RecursiveInterrupt,
-        SHCNE_DISKEVENTS | SHCNE_RENAMEFOLDER | SHCNE_RMDIR | SHCNE_MKDIR,
-        WM_USER_SHELLEVENT,
-        1,
-        &shcne);
+    shcne = { pidl, TRUE };
```

Comment:
Note: since this is C++ code you may consider moving the shcne variable declaration to there.

Reply:
Moved.